content/js/record.md

```diff
@@ -35,7 +35,13 @@ const Note = skygear.Record.extend('note');
 const Blog = skygear.Record.extend('blog');
 
 let note = new Note({ 'content': 'Hello World' });
-// { _id: "123456...", id: "note/123456...", recordType: "note", access: [Object object] }
+// note will be something like this:
+{
+  "_id": "123456...",
+  "id" : "note/123456...",
+  "recordType": "note",
+  // more fields
+}
```

Comment:
@rickmak should we talk about the internals here about a record in the guide? I am thinking if this is not necessary for an end user